README.md

```diff
@@ -42,15 +42,17 @@ You will also need to install:
       * You also need to install the `Command Line Tools` via Xcode. You can find this under the menu `Xcode -> Preferences -> Downloads`
       * This step will install `gcc` and the related toolchain containing `make`
   * On Windows:
-    * Visual C++ Build Environment:
-      * Option 1: Install [Visual C++ Build Tools](http://landinghub.visualstudio.com/visual-cpp-build-tools) using the **Default Install** option.
+    * Option 1: Install all the required tools and configurations using Microsoft's [windows-build-tools](https://github.com/felixrieseberg/windows-build-tools) using `npm install -g windows-build-tools` from an elevated PowerShell (run as Administrator).
```

Comment:
* Not using cmd: We'll spawn a little ps1 (invisible) in the background
* Not admin: You'll get a pretty error message (telling you to try again as admin) and an exit code 1. Installation is not attempted.